conf/application.conf

```diff
@@ -28,9 +28,7 @@ play.http.requestHandler = "uk.gov.hmrc.play.bootstrap.http.RequestHandler"
 application.router=prod.Routes
 play.i18n.langs = [ "en", "cy"]
 
-play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 www.google-analytics.com ajax.googleapis.com cdn.augur.io analytics.analytics-egain.com stats.g.doubleclick.net data: www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com"
-google-tag-manager.id-available = true
-google-tag-manager.id = "GTM-NDJKHWK"
+play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:9000 localhost:9032 localhost:12345  www.google-analytics.com ajax.googleapis.com cdn.augur.io analytics.analytics-egain.com stats.g.doubleclick.net data: www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com"
```

Comment:
Why `localhost:12345` ?

Reply:
For local testing. https://confluence.tools.tax.service.gov.uk/display/PLATUI/Integrating+with+tracking+consent+-+a+guide+for+service+teams